{{cookiecutter.profile_name}}/config.yaml

```diff
@@ -1,6 +1,6 @@
 restart-times: 3
 jobscript: "slurm-jobscript.sh"
-cluster: "slurm-submit.py"
+cluster: "slurm-submit.py --cluster '{cookiecutter.cluster_name}' --status_attempts 20"
```

Comment:
These arguments should be passed to slurm-status.py, no? In any case, with proper defaults setup in slurm-status.py itself, the options would be superfluous here so I prefer leaving the config file as it is.